app/src/main/res/drawable/recognizable_voice_error.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="oval"
+    >
+    <solid android:color="@android:color/white" />
```

Comment:
Using the Android white is probably pretty safe, but you never know. In general, it's better to be declarative and use your own resources instead of the Android SDK's. You never know what weird stuff can happen and Google developers can push bugs too. I remember Android O made the default accent color magenta or something, so a lot of apps started showing magenta everywhere.
```suggestion

    <solid android:color="color/white" />
```